config.py

```
@@ -63,0 +1,272 @@
+    (('email', 'sender'), 'str'),      # Could be enhanced for actual email format
+    (('email', 'smtp_server'), 'str'),
+    (('email', 'smtp_port'), 'int'),
+    (('email', 'smtp_username'), 'str'),
+    (('email', 'smtp_password'), 'str'), # Presence checked, value redacted
+    (('paths', 'download_dir'), 'str'), # Default is 'downloads' in other modules
+]
+
+# Substrings to identify keys that should have their values redacted in logs
+SECRET_KEY_SUBSTRINGS = [
+    "password", "token", "secret", "passwd", 
+    "smtp_user", "smtp_pass", "api_key" # Added common term
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)